package-deploy.yaml

```diff
@@ -0,0 +1,129 @@
+tasks:
+  - task: docker.Version
+    configFilePath: scripts/configure_dev-deps.sh
+
+  - task: shell.docker.Ensure
+    name: deb
+    image: algorand/mule-linux-ubuntu
+    version: '{{ docker.Version.outputs.version }}'
+    dockerFilePath: docker/build/docker.ubuntu.Dockerfile
+    dependencies: docker.Version
+
+  - task: shell.docker.Ensure
+    name: rpm
+    image: algorand/mule-linux-centos
+    version: '{{ docker.Version.outputs.version }}'
+    dockerFilePath: docker/build/mule.go.centos.Dockerfile
+    dependencies: docker.Version
+
+  - task: docker.Make
+    name: generate-releases-page
+    docker:
+      image: algorand/mule-linux-ubuntu
+      version: '{{ docker.Version.outputs.version }}'
+      workDir: /projects/go-algorand
+      env: [
+        AWS_ACCESS_KEY_ID=$AWS_ACCESS_KEY_ID,
+        AWS_SECRET_ACCESS_KEY=$AWS_SECRET_ACCESS_KEY
+      ]
+    target: mule-deploy-generate_releases_page WORKDIR=/projects/go-algorand
+
+  - task: docker.Make
+    name: deb
+    docker:
+      image: algorand/mule-linux-ubuntu
+      version: '{{ docker.Version.outputs.version }}'
+      workDir: /projects/go-algorand
+      env: [
+        AWS_ACCESS_KEY_ID=$AWS_ACCESS_KEY_ID,
+        AWS_SECRET_ACCESS_KEY=$AWS_SECRET_ACCESS_KEY
+      ]
+      volumes: [
+        $HOME/packages:/root/packages,
+        $XDG_RUNTIME_DIR/gnupg/S.gpg-agent:/root/.gnupg/S.gpg-agent,
```

Comment:
Is this expecting any host this task runs on to have some certain gpg configurations in this directory?

Reply:
Yes, the signing will need to be done before the task is run.